react-app/src/components/projects/ProjectsTable.tsx

```diff
@@ -72,13 +73,15 @@ const ProjectsTable = () => {
       flex: 1,
       valueFormatter: (date) => dateFormatter(date),
       maxWidth: 125,
+      type: 'date',
     },
     {
       field: 'UpdatedBy',
       headerName: 'Updated By',
       flex: 1,
       maxWidth: 150,
-      valueGetter: (user: User) => `${user?.FirstName ?? ''} ${user?.LastName ?? ''}`,
+      valueFormatter: (user: User) => `${user?.FirstName ?? ''} ${user?.LastName ?? ''}`,
+      type: 'date',
```

Comment:
I think this may be a mistake

Reply:
Pushed a change to revert this. Think I read updated on...